datasets/pl/finanse_sanctions/pl_finanse_sanctions.yml

```diff
@@ -33,13 +33,26 @@ publisher:
   official: true
 url: https://www.gov.pl/web/finanse/lista-osob-i-podmiotow-wobec-ktorych-stosuje-sie-szczegolne-srodki-ograniczajace-na-podstawie-art-118-ustawy-z-dnia-1-marca-2018-r-o-przeciwdzialaniu-praniu-pieniedzy-i-finansowaniu-terroryzmu
 data:
-  url: https://docs.google.com/spreadsheets/d/e/2PACX-1vR1QhgLSbLVzombbJu20IB1eJ-ORMazTPj7TCTHDJ-y_bo4dEoISAZgE3DL9c7wszmXEfY13L6P78GR/pub?gid=0&single=true&output=csv
-  format: CSV
+  url: https://www.gov.pl/attachment/56238b34-8a26-4431-a05a-e1d039f0defa
```

Comment:
if we get the data url from https://www.gov.pl/web/finanse/lista-osob-i-podmiotow-wobec-ktorych-stosuje-sie-szczegolne-srodki-ograniczajace-na-podstawie-art-118-ustawy-z-dnia-1-marca-2018-r-o-przeciwdzialaniu-praniu-pieniedzy-i-finansowaniu-terroryzmu, just make that the data url to avoid the data url going stale.